apps/wallet/src/ui/app/components/accounts/ZkLoginAccountWaringModal.tsx

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { zkProviderDataMap, type ZkProvider } from '_src/background/accounts/zk/providers';
+import { isZkAccountSerializedUI } from '_src/background/accounts/zk/ZkAccount';
+import { type MethodPayload } from '_src/shared/messaging/messages/payloads/MethodPayload';
+import {
+	Dialog,
+	DialogContent,
+	DialogDescription,
+	DialogFooter,
+	DialogHeader,
+	DialogTitle,
+} from '_src/ui/app/shared/Dialog';
+import { useMutation } from '@tanstack/react-query';
+import toast from 'react-hot-toast';
+
+import { useActiveAccount } from '../../hooks/useActiveAccount';
+import { useBackgroundClient } from '../../hooks/useBackgroundClient';
+import { Button } from '../../shared/ButtonUI';
+import { Link } from '../../shared/Link';
+
+const nameToProvider: Record<ZkProvider, string> = {
+	google: 'Google',
+	facebook: 'Facebook',
+	twitch: 'Twitch',
+};
+
+export function ZkLoginAccountWarningModal() {
+	const activeAccount = useActiveAccount();
+	const backgroundClient = useBackgroundClient();
+	const warningMutation = useMutation({
+		mutationKey: ['acknowledgeZkLoginWarning mutation'],
```

Comment:
while there's nothing wrong with this, the key reads kind of odd (e.g. I usually expect one word with kebab-case). Since we never set query option defaults in the wallet, we could just remove the key since it's effectively not doing anything 😆

Reply:
I will update it to kebab case